packages/app/src/app/pages/NewDashboard/Components/Breadcrumbs/index.tsx

```diff
@@ -1,43 +1,45 @@
 import React from 'react';
 import { Text, Link } from '@codesandbox/components';
 import { Link as LinkBase } from 'react-router-dom';
-import { dashboard } from '@codesandbox/common/lib/utils/url-generator';
 
-interface BreadcrumbProps {
-  path: string;
-  activeTeam: string;
-}
+export const Breadcrumbs = ({ param, repos }) => {
+  const params = param.split('/').length - 1;
 
-export const Breadcrumbs: React.FC<BreadcrumbProps> = ({
-  path,
-  activeTeam,
-}) => (
-  <Text marginBottom={1} block weight="bold" size={5}>
-    <Link
-      to={dashboard.allSandboxes('/', activeTeam)}
-      as={LinkBase}
-      variant={path && path.split('/').length ? 'muted' : 'body'}
-    >
-      All Sandboxes {path && ' / '}
-    </Link>
-    {path
-      ? path.split('/').map((p, i) => {
-          const partPath = path
-            .split('/')
-            .slice(0, i + 1)
-            .join('/');
+  const makeLink = (p: string, link: string) => {
```

Comment:
There was a bug in breadcrumbs for 2 folders deep, which is why I refactored this component. I think it would be better to base the changes in `Breadcrumbs` on the new version instead, I can definitely help with that!

Reply:
updated :)